### PR TITLE
Add NV41 documentation

### PR DESCRIPTION
**Pull request by mkopec**
Move NV41 documentation from GitLab.

Known issue links not changed yet (issues need to be moved to GitHub first)
Added common coreboot section with vboot signing procedure.

Signed-off-by: Michał Kopeć <michal.kopec@3mdeb.com>

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -85,6 +85,14 @@ nav:
             - 'Building manual': variants/talos_2/building-manual.md
             - 'Installation manual': variants/talos_2/installation-manual.md
             - 'Hardware configuration matrix': variants/talos_2/hardware-matrix.md
+        - 'Clevo NV41':
+            - 'Releases': variants/clevo_nv41/releases.md
+            - 'Building': variants/clevo_nv41/building.md
+            - 'EC firmware update': variants/clevo_nv41/ec_update.md
+            - 'Flashing with external programmer': variants/clevo_nv41/flashing_external.md
+            - 'Flashing with internal programmer': variants/clevo_nv41/flashing_internal.md
+            - 'Hardware Configuration Matrix' : variants/clevo_nv41/testing_configuration.md
```
Comment (miczyg1) [inline mkdocs.yml:94]: some rows have a space before colon, some not. It seems that syntax should be without spaces
```suggestion
            - 'Hardware Configuration Matrix': variants/clevo_nv41/testing_configuration.md
```

Please apply to other rows as well

Comment (mkopec) [reply mkdocs.yml]: Fixed

File: docs/common-coreboot-docs/vboot_signing.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## Preparation
+
+Navigate to the coreboot tree.
+
+## Generating keys
+
+Start by updating git submodules:
+
+        git submodule update --init --checkout
```
Comment (miczyg1) [inline docs/common-coreboot-docs/vboot_signing.md:19]: No code blocks?

File: docs/variants/clevo_nv41/flashing_internal.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+## Intro
+
+Flashing coreboot can be done from Linux using flashrom with the internal
+programmer. This document describes the process of building, installing and
+running flashrom on Ubuntu 20.04.
+
+## Pre-setup
+
+Ensure that the EC is running firmware 1.07.05. Follow the steps outlined in
+[EC firmware update](../ec_update)
```
[the remark this answers is not in the document]
Comment (mkopec) [reply docs/variants/clevo_nv41/flashing_internal.md:12]: These links work on my local machine (tested with `mkdocs serve`)

File: docs/common-coreboot-docs/vboot_signing.md (hunk 2 above)
Comment (mkopec) [reply docs/common-coreboot-docs/vboot_signing.md:19]: Double indent is also displayed in a code block, but i can change it to ``` for clarity

Comment (mkopec) [reply docs/common-coreboot-docs/vboot_signing.md]: Updated

File: docs/variants/clevo_nv41/custom_logo.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+Follow the steps specified in [the building instructions](../building)
+
+## Procedure
+
+The script `build.sh` located in the coreboot directory can be used
+to replace the logo. To replace the logo, run the following command:
+
+```bash
+./build.sh build -l path/to/logo.bmp
```
Comment (macpijan) [inline docs/variants/clevo_nv41/custom_logo.md:29]: @mkopec Sometimes we use some prompt before commands, sometimes not. Let's be consistent across all Dasharo docs.

Comment (mkopec) [reply docs/variants/clevo_nv41/custom_logo.md]: Added prompts before all commands

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**